### DISCUSSION TITLE
Change nightmode color?

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#1706] [Q&A]
Is there a option or way to change the nightmode color from orange to something else? Maybe a color picker dialog or config.ini string could be made?

**Comment by gzotti** [comment]
No. Red or better, orange, are the colors which do the least damage to dark-adapted eyes.

**Comment by axd1967** [comment]
@Keatah if you have a color blindness issue, see https://sunshinecoastastronomy.wordpress.com/2016/03/02/whats-the-best-night-vision-lighting-for-the-color-blind/
